modules/router/src/Router.ts

```diff
@@ -461,4 +470,161 @@ export default class ConduitDefaultRouter extends ManagedModule<Config> {
     }
     ConduitGrpcSdk.Logger.log('Recovered routes');
   }
+
+  async patchMiddleware(
```

Comment:
Maybe rename this to `patchRouteMiddlewares` or sth that would convey we're actually affecting the route's middlewares.